source/setup/nao_setup.rst

```diff
@@ -4,9 +4,12 @@ NAO Setup
 
 .. warning::
     Instructions here are not complete yet.
-
-Follow these instructions to prepare Naos, that are brand new or have been
-factory reset, to run the rUNSWift codebase.
+    
+.. note::
+    Follow these instructions to prepare Naos that are brand new or have been
+    factory reset, to run the rUNSWift codebase. 
```

Comment:
this needs updating

Reply:
how so?

Reply:
because you don't need to download an opn and flash the robot on a robot that has been factory reset

Reply:
you just need nao_sync -s if it has been factory reset

Reply:
it's funny.  this is the line you updated at the start, and had you updated to say this page is only for new robots, this pr probably would have been merged.